app/controllers/documents/requested_documents_later_controller.rb

```diff
@@ -1,7 +1,7 @@
 module Documents
   class RequestedDocumentsLaterController < DocumentUploadQuestionController
     before_action :handle_session, only: :edit
-    skip_before_action :require_sign_in
+    skip_before_action :require_intake
```

Comment:
we should remove this line. This controller skipped sign in but does require an intake.

Reply:
@bengolder - I am pretty sure we need to skip the `require_intake` behavior of this controller, but not the `SendRequestedDocumentsLaterController`, since this controller  performs its own management of `session[:intake_id]`, and thus `current_intake` will be defined by the `handle_session` before_action, and will be available in the `Send...` controller.